tools/applicationinsights-web-snippet/src/type.ts

```diff
@@ -0,0 +1,63 @@
+
+import { IConfig } from "@microsoft/applicationinsights-common";
```

Comment:
This should be

`import { IConfiguration } from "@microsoft/applicationinsights-web";